cg/services/run_devices/run_names/service.py

```diff
@@ -0,0 +1,12 @@
+from abc import ABC, abstractmethod
+
+
+class RunDirectoryNamesService(ABC):
+
+    def __init__(self, run_directory: str):
+        self.run_directory = run_directory
+
+    @abstractmethod
+    def get_run_names(self) -> list[str]:
+        """Get all the run names from a run directory."""
```

Comment:
```suggestion
        """Get all the run directory names from a data directory."""
```

Reply:
What is the difference? Is the run directory name the run name + well?

Reply:
true, dismissing